client/checkout/classic/upe.js

```diff
@@ -288,6 +296,11 @@ jQuery( function ( $ ) {
 			fillWith( 'billing_postcode', 'postal_code' );
 			fillWith( 'billing_country', 'country' );
 		} );
+
+		// Display StripeLink button if email field is prefilled.
+		if ( '' !== $( '#billing_email' ).val() ) {
+			$( '.wcpay-stripelink-modal-trigger' ).show();
```

Comment:
I think we could remove the button when the user removes the email and let the email field empty, since it makes no sense to fire the Link event. 
Or maybe remove it when the user makes a change in the email field. 
What's your opinion on this ?